wasm/src/debug_service.rs

```diff
@@ -4,33 +4,26 @@
 use std::str::FromStr;
 
 use qsc::fir::StmtId;
-use qsc::interpret::stateful::Interpreter;
+use qsc::interpret::stateful::Debugger;
 use qsc::interpret::{stateful, StepAction, StepResult};
-use qsc::{fmt_complex, target::Profile, PackageType, SourceMap};
+use qsc::{fmt_complex, target::Profile};
 
 use crate::{get_source_map, serializable_type, CallbackReceiver};
 use serde::{Deserialize, Serialize};
 use serde_json::json;
 use wasm_bindgen::prelude::*;
 
 #[wasm_bindgen]
+#[derive(Default)]
 pub struct DebugService {
-    interpreter: Interpreter,
+    debugger: Option<Debugger>,
 }
 
 #[wasm_bindgen]
 impl DebugService {
     #[wasm_bindgen(constructor)]
     pub fn new() -> Self {
-        Self {
-            interpreter: Interpreter::new(
-                false,
-                SourceMap::default(),
-                PackageType::Lib,
-                Profile::Unrestricted.into(),
-            )
-            .expect("Couldn't create interpreter"),
-        }
+        Self::default()
```

Comment:
The reason I removed the initialization here is that the `Interpreter` object created here isn't actually good for anything. `get_breakpoints` will return an empty list and any attempt to execute code will throw a "no entrypoint" error. We also never use it in practice. So I thought it's clearer to make it an optional field.